### PR TITLE
[acr dataplane] add changelog

### PR DESCRIPTION
**Pull request by adewaleo**

### CODE REVIEW

**Comment by adewaleo** [comment]
@chidozieononiwu @AlexGhiondea @weshaggard I have added the changelog. Hoping we can merge and release today....

File: sdk/containerregistry/Microsoft.Azure.ContainerRegistry/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Release History
+
+## 1.0.0-preview.1
```
Comment (chidozieononiwu) [inline sdk/containerregistry/Microsoft.Azure.ContainerRegistry/CHANGELOG.md:3]:
```suggestion
## 1.0.0-preview.1 (2020-04-06)
```

File: sdk/containerregistry/Microsoft.Azure.ContainerRegistry/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+- Require `System.IdentityModel.Tokens.Jwt >= 5.1.2` instead of `System.IdentityModel.Tokens.Jwt == 5.1.2`.
+
```
Comment (chidozieononiwu) [inline sdk/containerregistry/Microsoft.Azure.ContainerRegistry/CHANGELOG.md:6]: Consider adding some note here about "Updating package version to follow standard Azure-SDK-for-Net versioning guidelines"

**Review by chidozieononiwu — CHANGES_REQUESTED** [review]

**Review by AlexGhiondea — APPROVED** [review]

**Comment by adewaleo** [comment]
could someone merge?

**Review by chidozieononiwu — APPROVED** [review]

**Pull request merged**